FhirToDataLake/src/Microsoft.Health.AnalyticsConnector.Core/Jobs/FhirToDataLakeOrchestratorJob.cs

```diff
@@ -212,7 +247,142 @@ await foreach (FhirToDataLakeProcessingJobInputData input in inputs.WithCancella
             }
         }
 
+        private async Task<OrchestratorJobStatusEntity> InitializeJobStatusAsync(CancellationToken cancellationToken)
+        {
+            OrchestratorJobStatusEntity jobStatus = await _metadataStore.GetOrchestratorJobStatusAsync(_jobInfo.QueueType, _jobInfo.GroupId, _jobInfo.Id, cancellationToken);
+
+            if (jobStatus == null)
+            {
+                // Orchestrator job status entity is not exist, will create a new one.
+                var newJobStatus = new OrchestratorJobStatusEntity()
+                {
+                    PartitionKey = TableKeyProvider.JobStatusPartitionKey(_jobInfo.QueueType, Convert.ToInt32(JobType.Orchestrator)),
+                    RowKey = TableKeyProvider.JobStatusRowKey(_jobInfo.QueueType, Convert.ToInt32(JobType.Orchestrator), _jobInfo.GroupId, _jobInfo.Id),
+                    GroupId = _jobInfo.GroupId,
+                    StatisticResult = JsonConvert.SerializeObject(new FhirToDataLakeOrchestratorJobResult()),
```

Comment:
seems we need initialize statistic result from `_result` instead of a new `FhirToDataLakeOrchestratorJobResult` ?